packages/engine/src/utils.rs

```diff
@@ -46,6 +56,72 @@ enum OutputFormatter<T> {
     Compact(Format<fmt::format::Compact, T>),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, clap::ArgEnum)]
+pub enum OutputLocation {
+    StdOut,
+    StdErr,
+    File(PathBuf),
+}
+
+impl Display for OutputLocation {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        match self {
+            OutputLocation::StdOut => f.write_str("stdout"),
+            OutputLocation::StdErr => f.write_str("stderr"),
+            OutputLocation::File(path) => Display::fmt(&path.to_string_lossy(), f),
+        }
+    }
+}
+
+impl FromStr for OutputLocation {
+    type Err = Infallible;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "stdout" => Ok(Self::StdOut),
+            "stderr" => Ok(Self::StdErr),
+            _ => Ok(Self::File(PathBuf::from_str(s)?)),
+        }
+    }
+}
```

Comment:
Maybe we want to default to `stderr` and only if an output location is specify, it will be used as file? Otherwise files like `stderr` and `stdout` wouldn't be used (which isn't a big deal).

Reply:
I think for now it's fine as is. Maybe people don't like the idea of normal logs going to std _err_ and just want it in the standard out